owl/SOMA-HOME.owl

```diff
@@ -593,10 +593,299 @@
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#MilkPack -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#MilkPack">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Pack"/>
+        <rdfs:comment>A pack which holds, or at least is intended to usually hold, milk.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#NutellaJar -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#NutellaJar">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Jar"/>
+        <rdfs:comment>A jar which holds, or at least is intended to usually hold, Nutella.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#PastaBowl -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#PastaBowl">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Bowl"/>
+        <rdfs:comment>A bowl which holds, or at least is intended to usually hold, pasta.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#PepperShaker -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#PepperShaker">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Shaker"/>
+        <rdfs:comment>A shaker which holds, or at least is intended to usually hold, pepper.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#RaspberryJamJar -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#RaspberryJamJar">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#JamJar"/>
+        <rdfs:comment>A jar which holds, or at least is intended to usually hold, raspberry jam.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#SaladBowl -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#SaladBowl">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Bowl"/>
+        <rdfs:comment>A bowl which holds, or at least is intended to usually hold, salad.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#SaltShaker -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#SaltShaker">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Shaker"/>
+        <rdfs:comment>A shaker which holds, or at least is intended to usually hold, salt.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#Shaker -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#Shaker">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#DesignedContainer"/>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#SidePlate -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#SidePlate">
```

Comment:
It is possible if we can stick to some sort of hierarchy here for plates? We already have dinner plate then can we stick to such classification of plates where we attached the function they should serve in their name? I have never encounter term called "SidePlate". Maybe this can help? I see "BreadAndButterPlate" name suits best to our description here.
https://www.tableking.com.au/types-of-tableware-plates/

Reply:
So I measuerd the plate and it is 20cm in diameter. We actually just distinguish small and large plate.
So I think from your list **BreakfastPlate** would fit best, and I would be fine with that.

Reply:
okay then. Now, we can add ease-tsd concepts to SOMA-HOME.